docs/platform/release-notes/tizen-6-5-m1.md

```diff
@@ -0,0 +1,554 @@
+    - PropertyNotifySignal class has been removed.
+    - Property class of ItemView, Scorallable, Renderer, and Shader has been removed.
+    - LayerBehavior.Layer2D enum has been removed.
+    - Shadow class has been added.
+    - Theme class has been added.
+    - TimePeriod class has been added.
+    - Transition class has been added.
+    - ControlState class has been added.
+    - ViewStyle class has been added.
+    - AppBar class has been added.
+    - ContentPage class has been added.
+    - Navigator class has been added.
+    - Page class has been added.
+  - Tizen.Security
+    - TEEC has been deprecated.
+- Xamarin.Forms
+  - Xamarin.Forms 5.0.0, which is the latest stable version has been supported.
+
+#### Fixes
+
+- Tizen.Network.Bluetooth
+  - Marshaler coverting bugs have been fixed.
+  - Null reference bugs have been fixed.
+
+
+### Toolchain
+
+#### New and changed features
+
+- New architecture(Armv7hl) support
+  - Gcc: Armv7hl configuration has been newly added for hard float ABI.
+
+#### Fixes
+
+- Gcc/ Libsanitizer
+  - SIGSEGV in fopen64 interceptor has been fixed.
+- Glibc/ iconvdata (CVE-2019-25013)
+  - Buffer overrun in EUC-KR conversion module has been fixed.
+
+
+### Machine Learning
+
+#### New and changed features
+
+- ML / Web Inference API
+  - New API set, Pipeline and Single have been added.
+- Machine Learning Inference API update
+  - The package name has been changed from capi-nnstreamer to capi-machine-learning-inference. However, this is a platform-internal change, which does not affect application developers.
+  - The ability to register a tensor_if custom callback function in case the condition of tensor_if becomes complex has been added.
+  - The ability to open the model with various custom options such as flexible or fixed input dimensions has been added; i.e. ml_single_open_full().
+  - The ability to check whether the given element is registered or not has been added.
+  - The ability to invoke the model with pre-allocated output buffers, which is generally faster than normal invocation, has been added; i.e. ml_single_invoke_fast().
+- NNStreamer update
+  - NNStreamer has been upgraded from 1.6.0 to 1.7.1.
+  - The ability to create conditional branches (i.e. tensor_if) based on tensor values has been added. For example, developers may skip frames if there is no object detected with high confidence.
+  - The ability to support TensorFlow Lite 2 with GPU and NNAPI delegations along with gRPC/flatbuffer with sync and async mode has been added.
+  - The ability to designate priorities between multiple frameworks in case multiple frameworks support the same model file format (e.g., tensorflow-lite’s .tflite format) has been added.
+  - The ORC acceleration for tensor manipulation and subplugin auto detection based on the extension of the model file has been added.
+  - An ml_nnfw_type_e enumeration has been added to support pytorch and nntrainer.
+- Machine Learning Training API update

```

Comment:
```suggestion
- Machine Learning training API update

Reply:
It's more like Noun, Not verb. So.. I guess.. Training is better. 
like Machine Learning Training API, Machine Learning Inference API..

Reply:
sure thanks